centreon/www/front_src/src/CloudNotificationsConfiguration/Listing/index.tsx

```diff
@@ -36,13 +38,29 @@ const NotificationsListing = (): JSX.Element => {
   const [sorto, setSorto] = useAtom(sortOrderAtom);
   const [sortf, setSortf] = useAtom(sortFieldAtom);
   const [page, setPage] = useAtom(pageAtom);
+  const [reload, setReload] = useAtom(reloadAtom);
   const [isPannelOpen, setIsPannelOpen] = useAtom(isPanelOpenAtom);
   const panelWidth = useAtomValue(panelWidthStorageAtom);
   const setLimit = useSetAtom(limitAtom);
   const setEditedNotificationId = useSetAtom(EditedNotificationIdAtom);
   const setPanelMode = useSetAtom(panelModeAtom);
+  const setNotificationsNames = useSetAtom(notificationsNamesAtom);
 
-  const { loading, data: listingData } = useLoadingNotifications();
+  const { loading, data: listingData, refetch } = useLoadingNotifications();
+
+  useEffect(() => {
+    if (reload === true) {
```

Comment:
```suggestion
    if (reload) {
```

Reply:
Btw, why do you need a reload state to refetch a query? With react-query, you can just either invalidate the query or refetch the query

Reply:
However, the reload state can serve as an event detector. In this case, it checks if the submission was successful.

Reply:
what do you think @Thebarda ?

Reply:
and I will need it later for duplication and deletion

Reply:
Done! @Thebarda